Rubberduck.SourceControl/Rubberduck.SourceControl.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\LibGit2Sharp.0.22.0-pre20150516171636\build\net40\LibGit2Sharp.props" Condition="Exists('..\packages\LibGit2Sharp.0.22.0-pre20150516171636\build\net40\LibGit2Sharp.props')" />
+  <!--<Import Project="..\packages\LibGit2Sharp.0.22.0-pre20150516171636\build\net40\LibGit2Sharp.props" Condition="Exists('..\packages\LibGit2Sharp.0.22.0-pre20150516171636\build\net40\LibGit2Sharp.props')" />-->
```

Comment:
signpost

Reply:
This is a part I wanted to discuss. What are those ".props"? I haven't managed to find any documentation and I don't know what did you need them for, that's why I didn't want to get rid of them without asking. I suppose you don't mind them, but I'm still curious about docs in case you know of any.

Reply:
@rubberduck203 ??